src/main/java/com/exasol/dbbuilder/dialects/postgres/PostgreSqlImmediateDatabaseObjectWriter.java

```diff
@@ -0,0 +1,41 @@
+package com.exasol.dbbuilder.dialects.postgres;
+
+import java.sql.Connection;
+
+import com.exasol.dbbuilder.dialects.*;
+
+/**
+ * PostgreSQL {@link DatabaseObjectWriter}.
+ */
+public class PostgreSqlImmediateDatabaseObjectWriter extends AbstractImmediateDatabaseObjectWriter {
+
+    /**
+     * Create a new instance of {@link PostgreSqlImmediateDatabaseObjectWriter}.
+     * 
+     * @param connectionToPostgres connection to the PostgreSQL database.
+     */
+    public PostgreSqlImmediateDatabaseObjectWriter(final Connection connectionToPostgres) {
+        super(connectionToPostgres);
+    }
+
+    @Override
+    protected String getQuotedColumnName(final String columnName) {
+        return PostgreSqlIdentifier.of(columnName).quote();
```

Comment:
I would prefer to move this to an instance variable. Ideally remove the hardcoded dependency as well and use dependency injection for it so it can be set from the `PostgreSqlObjectFactory`

Reply:
what exactly?

Reply:
right, solved